SportsHub/UnitTests/Services/AuthenticationServiceTests.cs

```diff
@@ -13,36 +17,42 @@ public class AuthenticationServiceTests
     private readonly Mock<IUserService> _userService;
     private readonly Mock<IPasswordHasher> _passwordHasher;
     private readonly IAuthenticationService _authentication;
+    private IFixture _fixture;
+    private readonly IPasswordCheckResult _passwordCheckResult;
 
     public AuthenticationServiceTests()
     {
-        _userService = new Mock<IUserService>();
-        _passwordHasher = new Mock<IPasswordHasher>();
+        SetupFixture();
+        _userService = _fixture.Freeze<Mock<IUserService>>();
+        _passwordHasher = _fixture.Freeze<Mock<IPasswordHasher>>();
         _authentication = new AuthenticationService(_userService.Object, _passwordHasher.Object);
+        _passwordCheckResult = new PasswordCheckResult(true, false);
     }
 
-    [Fact]
-    public async Task Authenticate_WithUsername_ReturnUser()
+    //Not Working !!!
```

Comment:
Do you need this comment? Is it true what stated in it?

Reply:
No its old , probably forgot to delete it with all resolving of conflicts